src/test/java/guitests/utils/LocalDateFormatter.java

```diff
@@ -0,0 +1,11 @@
+package guitests.utils;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateFormatter {
+    static String toSimpleFormat(LocalDateTime ldt){
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
```

Comment:
Was thinking that maybe we should have a class that stores datetime string patterns. Another class of mine that would be using this would be ToDoCardStyleManager where I format the date labels for viewing